ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
We are not obfuscating each component.  We are storing the obfuscated ids of other objects in the compositeId -- those other objects are, specifically, the parent objects of the object to which the compositeId corresponds.